pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_assigning-metadata-to-a-module.adoc

```diff
@@ -0,0 +1,30 @@
+[id="assigning-metadata-to-a-module_{context}"]
+= Assigning metadata to a module
+
+Assign metadata to a module for search engine optimization (SEO). SEO improves the findability of modules.
+
+.Prerequisites
+
+* You are logged in.
+* The module to be assigned metadata has been imported to {ProductShortName}.
+
+.Procedure
+
+. Navigate to link:{LinkToSearchPage}[Search page].
+
+. Click the name of the module you want to assign metadata.
++
+If you do not see the module listed, search for the modules using the search box.
+
+. In the list of module versions, click the three vertical dots next to the version of the module you want to assign metadata.
+
+. Click *Edit metadata* to assign the following metadata: 
++
+* *Product Name* - Select the Product and its Version that the module should be published for.
+* *Document use case* - Select the verb that best describes the use case for the module.
+* *Vanity URL fragment* - Enter a URL for the module. This is optional.
+* *Search keywords* - Enter the words that, when searched by users, should bring up the module in the user's search. This is optional.
```

Comment:
```suggestion
* *Search keywords* - (Optional) Enter any words or terms that users might use when searching for this content.
```
It's important to make it clear at the beginning that the item is optional, and as concisely as possible.
Also, when we're referring to content users (not authors) as users, then I think it's helpful to avoid using our internal term "module". But considering that we're writing for authors, you can throw that comment in the trash.